### PR TITLE
Mention the continuous import fix in the changelog

### PR DESCRIPTION
**Pull request by tobim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Continuously importing events from a Zeek process with a low rate of
+  emitted events resulted in a long delay until the data would be included
+  in the result set of queries. This is because the import process would
+  buffer up to 10000 events before sending them to the server as a batch.
```
Comment (mavam) [inline CHANGELOG.md]:
```suggestion
  buffer up to 10,000 events before sending them to the server as a batch.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Continuously importing events from a Zeek process with a low rate of
```
Comment (mavam) [inline CHANGELOG.md]: Is this specific to the Zeek reader or to all text-based readers?

Comment (lava) [reply CHANGELOG.md]: Currently it's specific to zeek, although we should probably have a follow-up story to see if the same issue needs to be fixed for all our other readers as well.

Comment (mavam) [reply CHANGELOG.md]: Okay.

**Pull request merged**